libmscore/style.cpp

```diff
@@ -79,6 +79,7 @@ static const StyleType styleTypes[] { // page sizes and margins in points w/deci
       { Sid::pageUnits,               "pageUnits",                 0            }, // QPageLayout::Unit
       { Sid::pageOrientation,         "pageOrientation",           0            }, // QPageLayout::Orientation 
       { Sid::marginOddLeft,           "marginOddLeft",            10.0/INCH*PPI },
+      { Sid::pageOddRightMargin,      "marginOddLeft",            10.0/INCH*PPI },
```

Comment:
The second column should be "pageOddRightMargin".

Reply:
Thanks again.  I did the minimum to get it past Travis, which is my current focus.  I figured if it compiled I was OK.  But as I think about it now, while Travis is running, the mtests might fail, if they verify values.  I'll fix this and wait for this Travis to finish before committing and pushing again.
Regardless, this would be an issue at run-time in MuseScore because I failed to edit the string when I reinserted pageOddRightMargin.